src/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  padding: 0;
```

Comment:
Don't set for `*` selector any other rules than `box-sizing`. Move `padding` inside another selector if you need it.